src/main/java/me/profelements/dynatech/items/electric/AntigravityBubble.java

```diff
@@ -19,16 +20,11 @@
 
 import java.util.ArrayList;
 import java.util.Collection;
-import java.util.HashSet;
-import java.util.Iterator;
 import java.util.List;
-import java.util.Set;
-import java.util.UUID;
 
 public class AntigravityBubble extends AMachine {
```

Comment:
When you apply the feedback, pop your @author tag here making sure to first add ProfElements first to maintain the correct author list.